circle.yml

```diff
@@ -19,8 +19,7 @@ dependencies:
 
 test:
   override:
-    - echo "Skipped...Running our uitests on every pull request breaks"
-    - echo "them irregularly and they're badly maintained at the moment."
+    - echo "Beep Boop"
```

Comment:
Had no idea what to put here but I had to revert the changes and remove that we are skipping. More of a debug thing. 😄